dataland-frontend/src/components/pages/RequestData.vue

```diff
@@ -0,0 +1,304 @@
+                name="submit_request_button"
+                @click="handleSubmission"
+                :disabled="!selectedFile"
+              >
+                Submit
+              </PrimeButton>
+            </div>
+          </div>
+        </div>
+      </div>
+    </TheContent>
+  </AuthenticationWrapper>
+</template>
+
+<script lang="ts">
+import FileUpload, { FileUploadSelectEvent } from "primevue/fileupload";
+import Message from "primevue/message";
+import PrimeDialog from "primevue/dialog";
+import Checkbox from "primevue/checkbox";
+import PrimeButton from "primevue/button";
+import { defineComponent, inject, ref } from "vue";
+import { AxiosResponse } from "axios";
+import Keycloak from "keycloak-js";
+import { ApiClientProvider } from "@/services/ApiClients";
+import { InviteMetaInfoEntity } from "@clients/backend";
+import TheContent from "@/components/generics/TheContent.vue";
+import TheHeader from "@/components/generics/TheHeader.vue";
+import AuthenticationWrapper from "@/components/wrapper/AuthenticationWrapper.vue";
+import InfoCard from "@/components/general/InfoCard.vue";
+import ProgressBar from "@/components/general/ProgressBar.vue";
+import { assertDefined } from "@/utils/TypeScriptUtils";
+import { formatBytesUserFriendly, roundNumber } from "@/utils/NumberConversionUtils";
+import {
+  UPLOAD_FILE_SIZE_DISPLAY_DECIMALS,
+  EXCEL_TEMPLATE_FILE_NAME,
+  UPLOAD_MAX_FILE_SIZE_IN_BYTES,
+} from "@/utils/Constants";
+
+export default defineComponent({
+  name: "RequestData",
+  components: {
+    AuthenticationWrapper,
+    PrimeButton,
+    TheHeader,
+    TheContent,
+    InfoCard,
+    ProgressBar,
+    FileUpload,
+    FileSelectMessage: Message,
+    Checkbox,
+    PrimeDialog,
+  },
+  setup() {
+    return {
+      getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
+      searchBarAndFiltersContainer: ref<typeof FileUpload>(),
+    };
+  },
+  data() {
+    return {
+      uploadProgressInPercent: 0,
+      isInviteSuccessful: false,
+      inviteResultMessage: "No invite result message available.",
+      submissionFinished: false,
+      submissionInProgress: false,
+      pathToExcelTemplate: "/" + EXCEL_TEMPLATE_FILE_NAME,
+      maxFileSize: UPLOAD_MAX_FILE_SIZE_IN_BYTES,
+      selectedFile: null,
+      hideName: false,
+      displayModal: false,
+    };
+  },
+
+  computed: {
+    submissionProgressTitle() {
+      if (this.submissionFinished) {
+        if (this.isInviteSuccessful) {
+          return "Success";
+        } else {
+          return "Submission failed";
+        }
+      } else {
+        return "Submitting file";
+      }
+    },
+  },
+
+  methods: {
+    createNewRequest() {
+      this.$router.go();
+    },
+
+    clearSelection() {
+      this.selectedFile = null;
+    },
+
+    handleSelectFile(event: FileUploadSelectEvent) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
Warum gibt es hier so viele eslint-disable-next-line? In neuem Code sollten die eigentlich nicht auftauchen.

Reply:
Ich checke mal, welche disablements ich hier rausschmeißen/umgehen kann.

Was ich jedoch jetzt schon sagen kann, ist, dass wir vor Zugriffe auf "refs" in anderen components auch eslint disablen.
Mindestens an diesen Stellen sollten wir es hier also brauchen.

Reply:
Ich habe es jetzt so weit geschafft, dass "eslint-disable"-statements zumindest nur vor ref-Zugriffen stehen.
Ist das ok?
Das machen wir z.B. auch im SearchCompaniesForFrameworkData.vue Zeile 211, 217, 291, 326

Wir könnten nochmal recherchieren/experimentieren, wie/ob man dem Code zu verstehen geben könnte, dass dieses ref existiert, und auch die Methode bzw. Variable existiert, die aufgerufen wird.
Aber sollten wir das dann als Teil dieser Story sehen, schließlich betrifft das unseren gesamten Frontend Code?